.vcpkg/versions/f-/ff7tk.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I still feel like "f-" for a dirname is strange

Reply:
It's how vcpkg works unfortunately. We can't change that :(